### PR TITLE
Fix oncoprint sorting by custom case list

### PR DESCRIPTION
**Pull request by adamabeshouse**

### CODE REVIEW

**Comment by kalletlak** [comment]
@adamabeshouse which format are we using for caseIds 1. `study1:sample1+study1:sample2` or 2. `study1:sample1\nstudy1:sample2`?
I think when sending request from study view it is sent in 1st format. @zhx828 correct me if I am wrong

**Comment by adamabeshouse** [comment]
@kalletlak that seems to be true. I'm lost... is it just handled both ways everywhere?

**Comment by jjgao** [comment]
@adamabeshouse we should support blank characters as the separator, e.g. space, tab, new line.

**Comment by zhx828** [comment]
@adamabeshouse @kalletlak in the QueryBuilder, we use `study1:sample1+study1:sample2`

File: src/shared/components/query/QueryStore.ts [hunk 1]
```diff
@@ -140,6 +140,8 @@ export class QueryStore
 				this.studiesHaveChangedSinceInitialization = true;
 			}
 		);
+
+		(window as any).queryStore = this;
```
Comment (alisman) [inline src/shared/components/query/QueryStore.ts]: queryStore is already exposed on window object under globalStores

**Comment by adamabeshouse** [comment]
@jjgao it automatically gets URL encoded, so that's not a problem. I think we should stick with the + everywhere

**Pull request merged**